cppe/CMakeLists.txt

```diff
@@ -62,35 +44,69 @@ add_library(cppe ${PYTHON_SHARED}
   core/electric_fields.cc
   core/math.cc
   core/pe_energies.cc
-  ${PYTHON_INTERFACE_SOURCES} ${PELIB_INTERFACE_SOURCES}
 )
 
+add_library(cppe
```

Comment:
Building the C++-only library could be disabled in case the Python interface target is to be built.

Reply:
Up to you. For conda I'll build two packages -- the lib and the pymod.

Reply:
I guess for now it's not a problem to have both...